Index: views/stats.ejs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/views/stats.ejs b/views/stats.ejs
--- a/views/stats.ejs	(revision 3f88ff637fad808e5b9d97fd976225f63d28f959)
+++ b/views/stats.ejs	(date 1656692872336)
@@ -682,6 +682,7 @@
         <% if(calculated.guild.level){ %><span class='stat-name'>Level: </span><span class='stat-value'><%= calculated.guild.level %></span><br><% } %>
         ">Guild: <%= calculated.guild.name %></button><% } %>
         <a href="https://plancke.io/hypixel/player/stats/<%= calculated.display_name %>" target="_blank" rel="noreferrer" class="additional-player-stat external-link">Plancke</a>
+        <a href="https://sky.coflnet.com/player/<%= calculated.uuid %>" target="_blank" rel="noreferrer" class="additional-player-stat external-link">AH history</a>
         <div id="additional_socials">
           <button data-copy-text="<%= calculated.uuid %>" class="copy-text additional-player-stat">Copy UUID</button>
           <% if('DISCORD' in calculated.social){ %><button data-copy-text="<%= calculated.social.DISCORD %>" class="additional-player-stat copy-text external-discord external-icon"><%= calculated.social.DISCORD %></button><% } %>
