Index: Dockerfile
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Dockerfile b/Dockerfile
--- a/Dockerfile	(revision 2898f7cc888c1361cbf14732ee061a6d57fb1fa6)
+++ b/Dockerfile	(date 1667152315812)
@@ -1,9 +1,10 @@
 # syntax=docker/dockerfile:1
-FROM node:current-slim
+FROM node:16-buster
 LABEL maintainer="SkyCrypt"
 RUN mkdir -p /usr/src/main
 WORKDIR '/usr/src/main'
 COPY . /usr/src/main
 RUN npm install -g pnpm
-RUN pnpm i && pnpm build
+RUN pnpm i
+RUN mkdir -p /usr/src/main/cache/json
 EXPOSE 32464
+CMD pnpm run start
